### ISSUE TITLE
`carthage update` fails due to ENABLE_TESTABILTY not being set

### ISSUE THREAD

**Issue by raj-room** [artman/Signals#15]
With Signals 2.1.0 and Xcode 7.0.1, `carthage update` fails with this error:
```
Tests.swift:11:18: error: module 'Signals' was not compiled for testing
A shell task failed with exit code 65:
```

This can be fixed by setting ENABLE_TESTABILITY to YES in the project file for release builds.

If it helps, I can send a PR with this change. Thanks!

**Comment by artman** [comment]
Thanks, feel free to send a PR!

**Comment by raj-room** [comment]
Sent a PR, so closing this issue.
See https://github.com/artman/Signals/pull/16

**Comment by artman** [comment]
Thanks, I published version 2.1.1 which should now work with carthage.
